app/jobs/dradis/plugins/csv/mapping_import_job.rb

```diff
@@ -0,0 +1,82 @@
+require 'csv'
+
+module Dradis::Plugins::CSV
+  class MappingImportJob < ApplicationJob
+    queue_as :dradis_project
+
+    def perform(file:, identifier_col_index:, mappings:, project_id:, uid:)
+      @logger = Log.new(uid: uid)
+      @logger.write { "Job id is #{job_id}." }
+      @logger.write { 'Worker process starting background task.' }
+
+      @file = file
+      @identifier_col_index = identifier_col_index
```

Comment:
```suggestion
      @identifier_col_index = identifier_col_index.to_i
```
So in the class, we don't have to typecast it anymore.